manifests/agent.pp

```diff
@@ -272,7 +273,7 @@
   $tlsservercertsubject                   = $zabbix::params::agent_tlsservercertsubject,
   String $agent_config_owner              = $zabbix::params::agent_config_owner,
   String $agent_config_group              = $zabbix::params::agent_config_group,
-  Boolean $manage_selinux                 = $zabbix::params::manage_selinux,
+  Optional[Boolean] $manage_selinux       = $zabbix::params::manage_selinux,
```

Comment:
Why did you make this optional? Wouldn't it make more sense to default to false on Windows?

Reply:
I made this undef because the fact $facts['selinux'] is undef on windows. I could move manage_selinux to every OS in the case block, but this just seems more confusing to me.

Reply:
oh crap. It seems to always be true/false on Linux. I propose the following change for the params.pp:

```puppet
$manage_selinux = $facts['selinux'] ? {
  true     => true,
  default => false,
}
```